immuni_exposure_ingestion/helpers/risk_level.py

```diff
@@ -64,3 +65,29 @@ def extract_keys_with_risk_level_from_upload(upload: Upload) -> Iterable[Tempora
     )
 
     return keys_at_risk_filtered
+
+
+def extract_keys_with_risk_level_from_upload_eu(upload: UploadEu) -> Iterable[TemporaryExposureKey]:
```

Comment:
If both Upload and UploadEU have a common superclass (e.g., `UploadBase`), then here we can have a single function without the need of duplicating it.

Reply:
We were wrong in naming the function, we changed it to `set_highest_risk_level_from_upload`.
For the Eu upload Teks we just need to set the transmission_risk_level to "highest"